### DISCUSSION TITLE
Add a playground

### DISCUSSION THREAD

**Discussion by yuusheng** [wakujs/waku#906] [Ideas]
Should we have a playground or something more convenient for us to debug?

Currently, examples in the examples folder are using the hardcode waku version. Whenever we want to test our code, we can only change some example's package.json to "waku": "workspace:*", and run `pnpm i` in root dir. We should have a playground folder that uses the workspace waku source code. And every pr should not change playground code, to have a better DX.

**Comment by dai-shi** [comment]
The hardcoded version is same as the workspace version (intentionally), so it means the same as `"workspace: *"`. So, you should be able to use examples.

**Reply by yuusheng** [reply]
But when you change some source code, the hardcoded version is no longer the newest code.
